src/types/autocomplete.d.ts

```diff
@@ -47,6 +47,7 @@ export interface AutocompleteRequestType extends Record<string, any> {
   features: Partial<RequestFeature>;
   feature_variants: Partial<RequestFeatureVariant>;
   searchandized_items: Record<string, any>;
+  preFilterExpression?: FilterExpression;
```

Comment:
I think this is in the wrong place. 

This is the type for the request object returned in the reponse. I think this should be in AutocompleteParameters

Reply:
ooh nice catch. I got lost in all the type inheritance. I'll add it there but wouldn't we want the type here as well since it is something we return in the request object?

Reply:
Probably. Would need to snake case it. Didn't see it in the search and browse. So if you could add them there too that would be great.